README.md

```diff
@@ -1 +1,32 @@
-# vault-credential-rotator
\ No newline at end of file
+# vault-credential-rotator
+A tool used in conjunction with **HashiCorp Vault** (https://www.vaultproject.io/) to rotate access keys & secret access keys 
+for the AWS secrets engine within Vault.
```

Comment:
@troydieter Change this to state that it can support more than AWS access keys but at the moment only AWS support is implemented.  Something like that

Reply:
Resolved by adding roadmap. It's implied that additional secrets engines will be supported.